src/shared/services/form/form-value.service.ts

```diff
@@ -161,7 +161,7 @@ export class FormValueService {
   }
 
   sanitiseDynamicLists(caseFields: CaseField[], editForm: any): any {
-    return this.fieldTypeSanitiser.sanitiseLists(caseFields, editForm);
+    return this.fieldTypeSanitiser.sanitiseLists(caseFields, editForm['data']);
```

Comment:
Is `editForm.data` better instead of `editForm['data'] `

Reply:
Done